src/global.scss

```diff
@@ -244,22 +244,42 @@ h6 {
     color: white;
   }
 }
-.colorPallette {
+.color-picker-container {
```

Comment:
global is getting crowder. can you create an index.scss file for this color picker component ?